client/packages/system/src/Patient/DetailView/DetailView.tsx

```diff
@@ -119,6 +122,11 @@ export const PatientDetailView: FC = () => {
                 label={t('label.programs')}
                 tabIndex={-1}
               />
+              <Tab
+                value={Tabs.Encounters}
+                label={t('label.encounters')}
+                tabIndex={-1}
```

Comment:
shouldn't have the negative tab index - that's just for the default. It's an accident that it is on the programs one.. sorry!

Reply:
done